website/common/script/content/appearance/backgrounds.js

```diff
@@ -666,7 +671,8 @@ const releaseDates = {
   backgrounds012024: '2024-01-04T08:00-05:00',
   backgrounds022024: '2024-02-06T08:00-05:00',
   backgrounds032024: '2024-03-05T08:00-05:00',
-  backgrounds042024: '2023-04-04T00:00-04:00',
+  backgrounds042024: '2024-04-04T00:00-04:00',
+  backgrounds052024: '2024-05-07T00:00-04:00',
```

Comment:
Looks like this and the previous should have been 8AM
Copy-paste is no help when the previous entry is wrong!

Reply:
Fixed, update incoming.